Allow non-root users to at least see the help output.

http://bugs.gentoo.org/91724

--- src/shutdown.c
+++ src/shutdown.c
@@ -462,2 +462,3 @@
   		fprintf(stderr, "shutdown: you must be root to do that!\n");
+		usage();
   		exit(1);
